apps/web/screens/ServiceDetails/ServiceDetails.tsx

```diff
@@ -4,27 +4,94 @@ import { withMainLayout } from '@island.is/web/layouts/main'
 import getConfig from 'next/config'
 import { CustomNextError } from '@island.is/web/units/errors'
 
-const { publicRuntimeConfig } = getConfig()
+import { GetNamespaceQuery } from '@island.is/web/graphql/schema'
+import {
+  Query,
+  QueryGetApiServiceByIdArgs,
+  QueryGetNamespaceArgs,
+} from '@island.is/api/schema'
+import { GET_NAMESPACE_QUERY } from '../queries'
+import { GET_API_SERVICE_QUERY } from '../queries/ApiCatalogue'
+import { SubpageMainContent, ServiceInformation } from '../../components'
+import { useQuery } from '@apollo/client'
+import { SubpageLayout } from '../Layouts/Layouts'
+import SidebarLayout from '../Layouts/SidebarLayout'
+import { LoadingIcon } from '@island.is/island-ui/core'
 
-/* TEMPORARY LAYOUT CREATED TO SCAFFOLD SERVICE DETAILS INTO THE WEB */
+const { publicRuntimeConfig } = getConfig()
 
 interface ServiceDetailsProps {
-  title: string
+  //service: ApiService
+  serviceId: string
+  strings: GetNamespaceQuery['getNamespace']
 }
 
-const ServiceDetails: Screen<ServiceDetailsProps> = ({ title }) => {
+const ServiceDetails: Screen<ServiceDetailsProps> = ({
+  /*service,*/ serviceId,
+  strings,
+}) => {
   const { disableApiCatalog: disablePage } = publicRuntimeConfig
 
   if (disablePage === 'true') {
     throw new CustomNextError(404, 'Not found')
   }
 
-  return <h1>{title}</h1>
+  const { data, loading, error } = useQuery<Query, QueryGetApiServiceByIdArgs>(
+    GET_API_SERVICE_QUERY,
+    {
+      variables: {
+        input: {
+          id: serviceId,
+        },
+      },
+    },
+  )
+
+  return (
+    <SubpageLayout
+      main={
+        <SidebarLayout
+          sidebarContent={<>Navigation menu will be displayed here</>}
```

Comment:
Should this be merged into main like this?

Reply:
Good question, do we want boilerplate code on `main`?